src/components/Board/SymbolSearch/SymbolSearch.component.js

```diff
@@ -446,6 +482,19 @@ export class SymbolSearch extends PureComponent {
         <SymbolNotFound />
       ) : null;
 
+    const inclusivityOptions = this.showInclusivityOptions ? (
+      <div>
```

Comment:
As users can see this options with other picto banks we can add something here to explain that this is only for ARASAAC pictohgrams

Reply:
Great!